blackbox_repository/conversion_scripts/scripts/fcnet_import.py

```diff
@@ -80,9 +86,9 @@ def save_objective_values_helper(name, values):
     # (n_hps, n_seeds, n_epochs)
     # todo utilize expand dim instead of reshape
     epochs = np.repeat(np.arange(1, 101).reshape(1, -1), n_hps * n_seeds, axis=0).reshape(n_hps, n_seeds, -1)
-    runtime_per_epoch = (100 / epochs) * runtime.reshape((n_hps, n_seeds, 1))
+    elapsed_time = (epochs / 100) * runtime.reshape((n_hps, n_seeds, 1))
     
-    save_objective_values_helper('runtime', runtime_per_epoch)
+    save_objective_values_helper('elapsed_time', elapsed_time)
```

Comment:
Could we keep the original name `runtime`? The reasoning was to import datasets as close as possible from the original source and modify it only downstream.

Reply:
Hi David, the original field 'runtime' is runtime for the whole training job. The script converts this to "elapsed time".

In the NB201 benchmark, metric_runtime is runtime for the given epoch, and you need to say that towards the back-end. I am afraid if we now call what is elapsed time (from start to end of epoch) 'runtime' here, this will be very confusing.

Reply:
Ok thanks, I agree.